e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -71,5 +71,22 @@ context('When fee record correction feature flag is enabled', () => {
 
       cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}`));
     });
+
+    describe('when user clicks back on the create record correction request screen', () => {
+      it('should return to premium payments tab with the checkbox selected', () => {
+        premiumPaymentsTab.premiumPaymentsTable
+          .checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status)
+          .click();
+
+        premiumPaymentsTab.createRecordCorrectionRequestButton().click();
+
+        backLink().click();
+
+        cy.url().should('eq', relative(`/utilisation-reports/${reportId}?selectedFeeRecordIds=${feeRecordAtToDoStatus.id}`));
+        premiumPaymentsTab.premiumPaymentsTable
```

Comment:
Can we add an empty line after the URL assertion please, finding a little hard to read